### ISSUE TITLE
Cant run: Compute grads of eps on meta validation data

### ISSUE THREAD

**Issue by lcruzcas** [TinfoilHat0/Learning-to-Reweight-Examples-for-Robust-Deep-Learning-with-PyTorch-Higher#1]
eps_grads = torch.autograd.grad(meta_val_loss, eps, allow_unused=True)[0].detach()
'NoneType' object has no attribute 'detach'

**Comment by bit-frozen** [comment]
same problem, can anyone tell me what is the problem and how to solve it, thanks~

**Comment by Andrew-wong-ty** [comment]
It works for me.  My torch version is 1.10.1
